ci/generate-markdown/generate-packaging.sh

```diff
@@ -183,13 +183,15 @@ cc_binary(
 
 | Library | Minimum version | Description |
 | ------- | --------------: | ----------- |
-| [gRPC][gRPC-gh] | 1.16.x | gRPC++ for Cloud Bigtable |
+| [Abseil][abseil-gh] | 20200923, Patch 3 | Abseil C++ common library |
+| [gRPC][gRPC-gh] | 1.35.x | gRPC++ for Cloud Bigtable |
```

Comment:
nit: it is not just for Cloud Bigtable anymore?

Reply:
Good point. I stole the description "An RPC library and framework" from gRPC's github page.